system-addon/content-src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage} = require("react-intl");
+const {actionTypes, actionCreators: ac} = require("common/Actions.jsm");
+
+/**
+ * Confirmation Dialog component.
+ * One primary action button, one cancel button.
+ *
+ * Content displayed is controlled by `data` prop the component receives.
+ * Example:
+ * data: {
+ *   // Any sort of data needed to be passed around by actions.
+ *   payload: site.url,
+ *   // Primary button SendToMain action.
```

Comment:
Maybe `"The action dispatched to the main process if the user clicks the confirmation button"`